### PR TITLE
Update BT automation to work with the new project for StatVar mode.

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

File: bigtable_automation/gcf/bt_trigger.go [hunk 1]
```diff
@@ -62,29 +65,49 @@ type environment struct {
...
-	// CurrentEnv defaults to prod, and exported for overriding in `cmd/main.go`.
-	CurrentEnv = prod
+	// CurrentEnv defaults to prodOld, and exported for overriding in
+	// `cmd/main.go`.
+	CurrentEnv = prodOld
 
 	envs = map[string]*environment{
-		prod: &environment{
+		prodOld: &environment{
+			btProjectID:			"google.com:datcom-store-dev",
```
Comment (shifucun) [inline bigtable_automation/gcf/bt_trigger.go:81]: nit: indent of the map value

File: bigtable_automation/gcf/bt_trigger.go [hunk 2]
```diff
@@ -62,29 +65,49 @@ type environment struct {
...
 			baseBTInstance:   "prophet-cache",
 			baseBTClusters:   []string{"prophet-cache-c1"},
 			baseBTNodesHigh:  300,
 			baseBTNodesLow:   20,
 			branchBTInstance: "prophet-branch-cache",
+			dataBucket:       "prophet_cache",
 			controlBucket:    "automation_control",
 		},
+		prodNew: &environment{
+			btProjectID:			"datcom-store",
```
Comment (shifucun) [inline bigtable_automation/gcf/bt_trigger.go:92]: nit: indent of the map value

**Review by shifucun — DISMISSED** [review]

File: bigtable_automation/gcf/bt_trigger.go (hunk 1 above)
Comment (pradh) [reply bigtable_automation/gcf/bt_trigger.go]: Ran gofmt.

File: bigtable_automation/gcf/bt_trigger.go (hunk 2 above)
Comment (pradh) [reply bigtable_automation/gcf/bt_trigger.go]: Ran gofmt

**Comment by pradh** [comment]
Thanks for the review!

**Review by shifucun — APPROVED** [review]

**Pull request merged**